src/render.js

```diff
@@ -43,10 +65,20 @@ const toggleSettingsVisibility = () => {
     settingsDiv.style.display = 'none';
     if (reportHTML) {
       // If there is a report, un-hide it.
-      document.getElementById('appBody').classList.remove('empty');
+      appBody.classList.remove('empty');
       document.getElementById('output').style.display = 'block';
     }
   }
+
+  if (darkmode && !darkmode.isActivated()) {
+    const darkModeReady = document.getElementsByClassName('darkmode-background');
+    console.log(darkModeReady.length);
```

Comment:
remove console logs?

Reply:
I'll close this PR and create another one. I'm having some issue when I tried to do what you suggested - putting the checkbox inside the generator's side nav setting section. I'm seeing some inconsistency when I apply the darkmode-js. Essentially, if the app doesn't have any cache, when you open a cucumber-feature folder, sometimes the darkmode is not applied on the first click (when the box is checked) you have to click it a second time. This creates an unmatched status of the 'Dark Mode' check box. However, after that if you load another cucumber-feature folder the check box works as expected. I'm not 100% sure what causes this to happen. My theory is it has something to do with npm cache.

Reply:
there's a cache that you can leverage for the local storage, i don't remember the exact module we use for persistence